.github/workflows/nightly.yml

```diff
@@ -370,3 +372,25 @@ jobs:
 
   SYCL:
     uses: ./.github/workflows/reusable_sycl.yml
+
+  alpine:
+    name: Alpine
+    strategy:
+      fail-fast: false
+      matrix:
+          build_type: [Debug, Release]
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+      with:
+        fetch-depth: 0
+
+    - name: Build Alpine image
+      run: |
+        docker build . -f .github/docker/alpine-3.20.Dockerfile -t alpine_image
+
+    - name: Run UMF build on Alpine image
```

Comment:
we could also run tests on the build (not sure if as part of `build.sh` or as a separate step here)

Reply:
Setting up the necessary dependencies on Alpine is not worth the effort (at least for the purpose of fixing this issue)